src/backend/api/Fusion.Resources.Domain/Queries/GetResourceAllocationRequests.cs

```diff
@@ -185,9 +191,10 @@ public async Task<QueryRangedList<QueryResourceAllocationRequest>> Handle(GetRes
                 if (!countOnly)
                 {
                     await AddWorkFlows(pagedQuery);
-                    await AddOrgPositions(pagedQuery, request.Expands);
                     await AddProposedPersons(pagedQuery);
                 }
+                
+                await AddOrgPositions(pagedQuery, request.Expands);
```

Comment:
Is there a reason we want to resolve the positions when only the count is relevant?

Reply:
Yes, we need the information on base positions when we try to filter out the requests that are relevant for that department.

Reply:
nvm, I changed another part of the implementation forgot to change this back. Will update